hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -36,6 +36,9 @@
 public class OverwriteWithLatestAvroPayload extends BaseAvroPayload
     implements HoodieRecordPayload<OverwriteWithLatestAvroPayload> {
 
+  public static final String DEFAULT_DELETE_FIELD = "_hoodie_is_deleted";
+  private String userDefinedDeleteField = DEFAULT_DELETE_FIELD;
```

Comment:
can we name it as "isDeletedField" or "isDeletedMarkerField". may or may not be user defined and hence.

Reply:
If the variable name  change to "isDeletedField" or "isDeletedMarkerField", it means the variable type should be boolean. Maybe the default value of userDefinedDeleteField change to null.

Reply:
no, I meant it as field name referring to column. Thats why suffixed with "field"